config/proto/targets.proto

```diff
@@ -27,22 +40,24 @@ message TargetsDef {
 
   TargetSystem target_system = 1;
   ConnectionType connection_type = 2;
-  ConnectionDetails connection_details = 3;
+  CephConnectionDetails ceph_connection_details = 3;
   string target_url = 5;  // URL for connecting to the the API of the target storage system.
   int64 total_space_allocated = 4;  // in MBs.
 }
 
 // REQUIRED, when using Ceph as the target system.
 // Comparable functionality is handled in the Go client library for GCS.
 message CephConnectionDetails {
-  int32 port = 1;  // Port for connecting to Ceph S3 API
-  
+  // REQUIRED for Ceph S3 API
+  // Port for connecting to Ceph S3 API
+  int32 port = 1;
+
   // REQUIRED for Ceph S3 API
   // GCS uses service account credentials instead.
   string api_key = 2;
 }
 
 // Targets allows the user to define multiple targets.
 message Targets {
-  repeated TargetDef targets = 1;
+  repeated TargetsDef targets = 1;
```

Comment:
Could you call this TargetDefinition? it's repeated, but it only includes one target, doesn't it?

Reply:
Yes, it only includes one target. I will change this.

Reply:
Changed.